op-node/chaincfg/chains.go

```diff
@@ -39,6 +39,7 @@ var Beta1 = rollup.Config{
 	L1SystemConfigAddress:  common.HexToAddress("0x686df068eaa71af78dadc1c427e35600e0fadac5"),
 }
 
+var goerliRegolithTime = uint64(1679079600)
```

Comment:
since we are going to need to do this multiple times, perhaps define a function

```
func u64Ptr(uint64) *uint64
```

Reply:
Yep, good idea.  Done.